### DISCUSSION TITLE
Can't resolve 'fast-crc32c'

### DISCUSSION THREAD

**Discussion by cammylamberth** [gladly-team/next-firebase-auth#225] [Q&A]
Hi, I'm trying to use with https://github.com/serverless-nextjs/serverless-next.js but getting Module not found Error: Can't resolve 'fast-crc32c'. Is this an issue with this or serverless-nextjs? 

Anyone any idea how to resolve this?

**Comment by uncvrd** [comment]
@cammylamberth any luck? I've discovered this has to do with the fact that `firebase-admin` is part of the node modules and expects a node environment. I'm looking for solutions but haven't found anything, seems strange and I feel like we're missing something. I've tried next-serverless and netlify
